plugins/outputs/seq/seq_test.go

```diff
@@ -0,0 +1,26 @@
+package seq
+
+import (
+	"testing"
+
+	"github.com/influxdata/telegraf/testutil"
+	"github.com/stretchr/testify/require"
+)
+
+func TestConnectAndWrite(t *testing.T) {
+	if testing.Short() {
+		t.Skip("Skipping integration test in short mode")
+	}
+
+	url := testutil.GetLocalHost() + ":5341"
```

Comment:
Need to add scheme (`http`) and don't forget to start the server somewhere in the tests too.

Reply:
Fixed. I refactored the test to use httptest.